src/control/common/test_mocks.go

```diff
@@ -50,7 +50,6 @@ func MockNamespacePB() *pb.NvmeNamespace {
 // multiple packages (message contains repeated namespace field).
 func MockControllerPB(fwRev string) *pb.NvmeController {
 	return &pb.NvmeController{
-		Id:        int32(12345),
```

Comment:
Why remove the ID? It seems to still appear in the output.

Reply:
no longer using the ID because they are not unique on a host. from the SPDK mailing list: "controller ID is only unique within an NVMe subsystem. In NVMe-oF the subsystems are set up in the configuration manually, but for physical devices each device is its own subsystem and will count its controllers from 0 typically." I've left it in the message definitions because we might have a use for it in future.